blocklytest/src/androidTest/java/com/google/blockly/model/BlockTestStrings.java

```diff
@@ -59,7 +59,7 @@
             + "    {"
             + "      \"type\": \"field_checkbox\","
             + "      \"name\": \"NAME\","
-            + "      \"checked\": true"
+            + "      \"checked\": TRUE"
```

Comment:
+1 to Andrew's comments.

Shouldn't we update Blockly Web to do a case-insensitive compare instead?